documentation/MIGRATION_GUIDE.md

```diff
@@ -55,42 +56,32 @@ For detailed information on the benefits of using the new authentication types,
 
 There are some minor changes in the error handling.
 
-- When there is an error in the SDK request, in the previous version (`services/**/mgmt/**`), the return value will all be non-nil, and you can get the raw HTTP response from the response value. In the latest version (`sdk/resourcemanager/**/arm**`), the first return value will be empty and you need to convert the error to the `azcore.HTTPResponse` interface to get the raw HTTP response. When the request is successful and there is no error returned, you can get the raw HTTP response in `RawResponse` property of the first return value.
+- When there is an error in the SDK request, in the previous version (`services/**/mgmt/**`), the return value will all be non-nil, and you can get the raw HTTP response from the response value. In the latest version (`sdk/resourcemanager/**/arm**`), the first return value will be empty and you need to convert the error to the `azcore.ResponseError` interface to get the raw HTTP response. When the request is successful and there is no error returned, you can get the raw HTTP response from request context.
 
 **Previous version (`services/**/mgmt/**`)**
 
 ```go
 resp, err := resourceGroupsClient.CreateOrUpdate(context.Background(), resourceGroupName, resourceGroupParameters)
 if err != nil {
-	log.Fatalf("Status code: %d", resp.Response.Response.StatusCode)
+    log.Fatalf("Status code: %d", resp.Response.Response.StatusCode)
```

Comment:
Needs an `errors.As()` to convert to `*azcore.ResponseError` first.

Reply:
This part of code uses track1 and the raw response can be fetched by operation resp directly.